### PR TITLE
Grid layout Card

### PR DESCRIPTION
**Pull request by Anisospina**
## Summary
Add CSS grid layout
New design

## Trello Card:
https://trello.com/c/kLq95Kp0/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: Maquetado/scss/tech.scss [hunk 1]
```diff
@@ -1,42 +1,56 @@
 .row {
-  display: flex;
+  display: grid;
   flex-wrap: wrap;
+  grid-template-columns: 34% 33% 33%;
```
Comment (jmtov) [inline Maquetado/scss/tech.scss]: You can use the `repeat()` function here.

File: Maquetado/scss/tech.scss [hunk 2]
```diff
@@ -1,42 +1,56 @@
...
+  grid-template-rows: 38% 38% auto;
```
Comment (jmtov) [inline Maquetado/scss/tech.scss]: Do you really need this property here?

File: Maquetado/scss/tech.scss (hunk 1 above)
Comment (jmtov) [reply Maquetado/scss/tech.scss]: Also, take a look at the `fr` unit: https://alligator.io/css/css-grid-layout-fr-unit/

File: Maquetado/scss/tech.scss [hunk 3]
```diff
@@ -1,42 +1,56 @@
...
   margin: 0 auto;
-  max-width: 1200px;
-  width: 100%;
+  width: 54%;
```
Comment (jmtov) [inline Maquetado/scss/tech.scss]: I'd recommend to set a `max-width` and leave the width to a 100%. Remember to test your layouts in different screen sizes!

File: Maquetado/scss/tech.scss [hunk 4]
```diff
@@ -1,42 +1,56 @@
...
+  &:nth-child(1) {
+    border-right: 2px solid $green;
+    grid-column-end: 1;
+    grid-column-start: 1;
+    grid-row-end: 3;
+    grid-row-start: 1;
+    padding-top: 13em;
+  }
+
+  &:nth-child(2) {
```
Comment (jmtov) [inline Maquetado/scss/tech.scss]: You don't need to define the position of every child within a grid layout. Well, at least not to achieve what the card defines.

File: Maquetado/scss/titles.scss [hunk 5]
```diff
@@ -1,10 +1,12 @@
...
   display: flex;
+  font-family: $font-roboto;
+  font-size: $text-big;
   height: 100px;
+  margin: 0 auto;
   padding-left: 60px;
-  color: $grey-dark;
-  font-size: $text-big;
-  font-family: $font-roboto;
+  width: 70%;
```
Comment (jmtov) [inline Maquetado/scss/titles.scss]: Do you really need to set the width on this element?

**Review by jmtov — CHANGES_REQUESTED** [review]
Nice job! There are some things that need to be adjusted. Please check out my comments! 😺

File: Maquetado/scss/tech.scss [hunk 6]
```diff
@@ -1,42 +1,28 @@
...
-  &:hover::before {
-    -ms-transform: translate(20px, 20px);
-    -webkit-transform: translate(20px, 20px); 
-    transform: translate(20px, 20px);
+  &:nth-child(1) {
+    border-right: 2px solid $green;
+    grid-column-end: 1;
+    grid-column-start: 1;
+    grid-row-end: 3;
+    grid-row-start: 1;
```
Comment (jmtov) [inline Maquetado/scss/tech.scss]: I forgot to tell you, but you can use the `grid-row` shorthand here. 😅

**Review by jmtov — CHANGES_REQUESTED** [review]

File: Maquetado/index.html [hunk 7]
```diff
@@ -90,11 +90,11 @@ <h3 class="title-tech">Django</h3>
...
   </div>
 
-  <footer>
+  <footer class="footer">
     <p class="text-footer">We Are Game Changers</p>
     <img class="img-logo" src="assets/wolox_logo_white.svg" alt="Logo Wolox white">
     <span class="copy">Copyright 2018. Wolox. All right reserved</span>
   </footer>
-
+
```
Comment (SKOLZ) [inline Maquetado/index.html]: delete the spaces here

File: Maquetado/scss/navbar.scss [hunk 8]
```diff
@@ -10,7 +10,7 @@
   position: fixed;
   width: 100%;
   z-index: 1;
-
+
```
Comment (SKOLZ) [inline Maquetado/scss/navbar.scss]: delete the spaces

**Review by SKOLZ — CHANGES_REQUESTED** [review]

File: Maquetado/scss/tech.scss [hunk 9]
```diff
@@ -1,42 +1,25 @@
...
-  }
-
-  &:hover::before {
-    -ms-transform: translate(20px, 20px);
-    -webkit-transform: translate(20px, 20px); 
-    transform: translate(20px, 20px);
+  &:first-child {
+    border-right: 2px solid $green;
+    grid-row: 1 / span 2;
+    padding-top: 13em;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: although this is not wrong, please use pixels to keep a fixed padding

**Review by SKOLZ — CHANGES_REQUESTED** [review]

**Review by jmtov — APPROVED** [review]

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**